src/global.css

```diff
@@ -88,4 +88,44 @@ a:hover {
 
 .cm-lineNumbers > .cm-gutterElement[err-line] {
 	background: #ecb2b2!important;
-}
\ No newline at end of file
+}
```

Comment:
It might be safer to move all sprikipedia css into its own file to ensure changes you make do not side-effect other preexisting pages...

Reply:
Ok! Will do